index.html

```diff
@@ -3,34 +3,37 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio Website: Ademola Adebayo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Portfolio: Ademola Adebayo</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="new.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Roboto:wght@100&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
 </head>
 <body>
-  <header class="main-head">
-    <nav class="toolbar">
-      <div class="upper-toolbar">
-        <div><h2>Smart City</h2></div>
-        <div class="nav-links">
-          <ul>
-            <li><a href=""></a>Portfolio</li>
-            <li><a href=""></a>About</li>
-            <li><a href=""></a>Contact</li>
-          </ul>
-          <a href=""><i class=" far fa-envelope fa-1em"></i></a> 
-        </div>
+  <header class="main">
+    <nav class="navbar">
+      <div class="nav-left">
+       
+          <h2>Smart City</h2>
+        
       </div>
-      <div class="top-toolbar">
-        <div><h2>Smart City</h2></div>
-        <a href=""><i class=" fas fa-bars fa-2x"></i></a>
+      <div class="nav-right">
+        <ul class="nav-links">
+          <li><a class="links" href="#portfolio">Portfolio</a></li>
+          <li><a class="links" href="#about">About</a></li>
+          <li><a class="links" href="#contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ] Good job working on some of the changes requested by the previous reviewer, however, the mobile menu still needs some changes. kindly make sure that your design (Fonts, colors, style, size, margins) matches the Figma template as much as possible.

Your design

![Capture](https://user-images.githubusercontent.com/85095702/163066945-188e352e-cbf1-4ef0-ad0d-f8aa60db08fa.PNG)

Figma template
![figma](https://user-images.githubusercontent.com/85095702/163066986-ed1ea245-6144-419a-87c7-478d7d2e3331.png)

Reply:
- [ ] Good job applying the correct styles to your mobile menu, however, there's something that we need to fix in order to pass to the next milestone. If you check your mobile version project, you will find out that the menu is being displayed properly, but if you try to scroll from right to left, you will see that your menu is causing a lot of horizontal scrolls, and, not only that, it is showing the menu on the side. This may be happening because you are playing a bit with the position of your menu container, but not with the display property, so you are placing your menu to the right side to try to hide it, but the display property is not changed, it is still visible. Here's an image so you can see what I mean:
![Captura de pantalla 2022-04-13 174424](https://user-images.githubusercontent.com/59100477/163218943-66fc6566-3f23-428e-965d-a89f5eeb9b85.jpg)

As mentioned, you can fix this by playing with the display property of your menu container, or either find another solution. So please, work on this issue.